pages/settings/profile.tsx

```diff
@@ -147,14 +160,46 @@ export default function Settings(props) {
                     First Day of Week
                   </label>
                   <div className="mt-1">
-                    <select
+                    <Select
                       id="weekStart"
                       value={selectedWeekStartDay}
-                      onChange={(e) => setSelectedWeekStartDay(e.target.value)}
-                      className="shadow-sm focus:ring-blue-500 focus:border-blue-500 mt-1 block w-full sm:text-sm border-gray-300 rounded-md">
-                      <option value="Sunday">Sunday</option>
-                      <option value="Monday">Monday</option>
-                    </select>
+                      onChange={setSelectedWeekStartDay}
+                      className="shadow-sm focus:ring-blue-500 focus:border-blue-500 mt-1 block w-full sm:text-sm border-gray-300 rounded-md"
+                      options={[
+                        { value:'Sunday', label:'Sunday' },
```

Comment:
Codacy has a fix for the issue: [Replace `'Sunday',·label:'Sunday'` with `·"Sunday",·label:·"Sunday"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
                        { value: "Sunday", label: "Sunday" },
```